### DISCUSSION TITLE
The swap on Eth Sepolia Reverts, even if enough funds

### DISCUSSION THREAD

**Discussion by ielboulo** [scaffold-eth/scaffold-eth-2#659] [Q&A]
The call of the function Swap_Eth_pay() reverts 
The reason unknown. 

The call was done with these arguments : 
- Eth Sepolia address : 0x99FCee8A75550a027Fdb674c96F2D7DA31C79fcD
- amount of ETH : 250000000000 wei ( 0,00000025 )
- Value  : ETH sepolia = 100000000000000000 wei ( 0.1 ETH) : the funds in the smart contract

**Comment by MatheusDaros** [comment]
Hello @ielboulo 

Thanks for opening this issue

Could you please confirm if the contract you're trying to call implements the function `Swap_Eth_pay`?
